azurerm/internal/services/apimanagement/api_management_resource.go

```diff
@@ -284,7 +293,63 @@ func resourceApiManagementService() *schema.Resource {
 							Default:  false,
 						},
 
+						// TODO: Remove after deprecation
 						"enable_triple_des_ciphers": {
+							Type:          schema.TypeBool,
+							Optional:      true,
+							Default:       false,
+							ConflictsWith: []string{"security.triple_des_ciphers_enabled"},
```

Comment:
Since this is a block, stored in state as a list with one element (having index `0`), this should be:
```suggestion
							ConflictsWith: []string{"security.0.triple_des_ciphers_enabled"},
```